src/JupyterLibrary/clients/jupyterlab/Shell.robot

```diff
@@ -22,9 +22,10 @@ Open JupyterLab
     Tag With JupyterLab Metadata    ${pageinfo tags}    clear=${clear}
 
 Wait for JupyterLab Splash Screen
+    [Arguments]    ${custom_timeout}=10s
```

Comment:
perhaps just `${timeout}`?

Note i also end up throwing the big `Wait Until Keyword Succeeds` at it, depending on the weight (and windows) of the situation.